### PR TITLE
mgr/dashboard_v2: Fix pylint errors exposed in python 2.7

### PR DESCRIPTION
**Pull request by rjfd**
This will make Jenkins green again :-)

Signed-off-by: Ricardo Dias <rdias@suse.com>

### CODE REVIEW

File: src/pybind/mgr/dashboard_v2/tests/test_nodb.py [hunk 1]
```diff
@@ -158,7 +158,7 @@ class NameInFilterSet(FilterSet):  # pylint: disable=E0602
             # pylint: disable=E0602
             name_in = CommaSeparatedValueFilter(name='name', lookup_type='in')
 
-            class Meta:
+            class Meta(object):
```
Comment (sebastian-philipp) [inline src/pybind/mgr/dashboard_v2/tests/test_nodb.py]: If you need to fix code in a skipped test, can you instead please remove this test altogether?

File: src/pybind/mgr/dashboard_v2/tests/test_nodb.py [hunk 2]
```diff
@@ -182,7 +182,7 @@ def get_all_objects(api_controller, query):
             my_dict = JsonField(base_type=list, primary_key=True)
 
         class DictFieldModelSerializer(NodbSerializer):  # pylint: disable=E0602
-            class Meta:
+            class Meta(object):
```
Comment (sebastian-philipp) [inline src/pybind/mgr/dashboard_v2/tests/test_nodb.py]: If you need to fix code in a skipped test, can you instead please remove this test altogether?

**Review by sebastian-philipp — CHANGES_REQUESTED** [review]

File: src/pybind/mgr/dashboard_v2/tests/test_nodb.py (hunk 1 above)
Comment (rjfd) [reply src/pybind/mgr/dashboard_v2/tests/test_nodb.py]: I will gladly remove it. Why were these tests added in the first place?

**Comment by rjfd** [comment]
@sebastian-philipp removed the skipped tests

File: src/pybind/mgr/dashboard_v2/models/cluster.py [hunk 3]
```diff
@@ -1,9 +1,18 @@
...
+
+from .nodb import NodbModel, CharField, JsonField, ValidationError, \
+                  bulk_attribute_setter
 from ..models.send_command_api import SendCommandApiMixin
 from ..tools import ExternalCommandError
-from .nodb import *
 from ..tools import RESTController  # pylint: disable=W0611
 
 
+logger = logging.getLogger(__name__)
```
Comment (sebastian-philipp) [inline src/pybind/mgr/dashboard_v2/models/cluster.py:13]: Afaik we should not use the python logger, but the logger exposed in the module. That's why I didn't add it in the first place.

**Review by sebastian-philipp — APPROVED** [review]

File: src/pybind/mgr/dashboard_v2/models/cluster.py (hunk 3 above)
Comment (rjfd) [reply src/pybind/mgr/dashboard_v2/models/cluster.py:13]: I agree. The problem is that in `nodb.py` was added a python logger, https://github.com/openattic/ceph/commit/7f1729c224afccf7870d81ace01c04ba31183807#diff-a80a3792a3b6f9c36a06ae4bf915ed70R18 , that was being imported in this file.
I added that line for now just to make sure that the code runs, but I'm expecting that someone will open a PR to change this logger for the manager module's logger.

**Pull request merged**